README.md

```diff
@@ -43,12 +43,20 @@ yarn install
 ```
 
 ## Testing
-Run a test network:
 ```
-./server.sh
+yarn test
 ```
 
-Then, run the test suite:
+If Ganache is already running on port 7545 (its default port), you must run:
 ```
-truffle test
+yarn test-genache
```

Comment:
```suggestion
yarn test-ganache
```

Reply:
OMG literally the most embarrassing thing ever :man_facepalming: